ports/zephyr/README.md

```diff
@@ -4,16 +4,20 @@
 This directory contains some proof-of-concept applications that show how to build and use CycloneDDS on [Zephyr RTOS](https://www.zephyrproject.org) for a NXP X-S32Z27X-DC board.
 
 
-Getting Started with Zephyr information can be found [here](https://docs.zephyrproject.org/3.3.0/develop/getting_started/index.html)
+Getting Started with Zephyr information can be found [here](https://docs.zephyrproject.org/latest/develop/getting_started/index.html)
+Documentation for the NXP X-S32Z27X-DC board can be found [here](https://docs.zephyrproject.org/latest/boards/arm/s32z270dc2_r52/doc/index.html)
 
+## Usage:
```

Comment:
There is usually no punctuation in titles

Reply:
Fixed!